topics/rediscli.md

```diff
@@ -345,6 +345,22 @@ key, like in the following example:
     127.0.0.1:6379> ZADD<TAB>
     127.0.0.1:6379> ZCARD<TAB>
 
+Once you've entered a valid command name at the prompt, its syntax will be
```

Comment:
Maybe:

"Once you've typed a Redis command name at the prompt, the CLI will display syntax hints.  This behavior can be turned on and off via the CLI preferences."

Reply:
Accepted